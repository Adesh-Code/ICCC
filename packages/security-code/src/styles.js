const styles = ({ tokens, primitives }) => ({
  "&.DhigSecurityCode": {
    width: "56px",
    height: "72px",
    borderTop: `${primitives.borderWidths.medium} solid ${primitives.colors.tint["slate-450"]}`,
    borderBottom: "none"
  },
  '&.DhigSecurityCode [class*="MuiInputBase-input"][class*="MuiInputBase-inputAdornedEnd"]':
    {
      marginRight: "0"
    },
  "&.DhigSecurityCode[class*='MuiInput-root']:hover:before": {
    height: "2px",
    backgroundColor: tokens.colors["fill-engage"],
    boxShadow: "none",
    borderBottom: `${primitives.borderWidths.medium} solid ${primitives.colors.primary.black}`
  },
  "&.DhigSecurityCode:hover": {
    boxShadow: "none"
  },
  '&.DhigSecurityCode[class*="MuiInputBase-root"]': {
    boxShadow: `0px 1px 0px 0px ${primitives.colors.primary.white}`
  },
  '&.DhigSecurityCode[class*="MuiInputBase-root"]:before': {
    borderBottom: `1px solid ${primitives.colors.primary.black}`
  },
  "&.DhigSecurityCode input:focus": {
    boxShadow: `0px 2px 0px 0px ${tokens.colors["ink-focus"]} inset, 0px 0px 0px 2px ${tokens.colors["ink-focus"]} inset`
  },
  '&.DhigSecurityCode[class*="MuiInputBase-root"]:not(.Mui-disabled)::before': {
    boxShadow: "none"
  },
  '&.DhigSecurityCode [class*="MuiInputBase-input"]:before': {
    width: "56px"
  },
  "&.DhigSecurityCode input": {
    flexGrow: "1"
  },
  '&.DhigSecurityCode [class*="MuiInputBase-input"]': {
    padding: "0",
    width: "56px",
    height: "72px",
    fontSize: `${primitives.fontSizeFallbacks["6"]}`,
    fontWeight: primitives.fontWeights.default
  },
  '&.DhigSecurityCode--dense [class*="MuiInputBase-input"]': {
    padding: "0",
    width: "32px",
    height: "40px",
    fontSize: `${primitives.fontSizeFallbacks["3"]}`,
    fontWeight: primitives.fontWeights.default
  },
  "&.DhigSecurityCode--dense": {
    width: "32px",
    height: "40px"
  },
  '&.DhigSecurityCode--error[class*="MuiInputBase-root"]:before': {
    borderBottom: `1px solid ${primitives.colors.accent.clay}`
  },
  "&.DhigSecurityCode--error, &.DhigSecurityCode:first-of-type.DhigSecurityCode--error, &.DhigSecurityCode:last-child.DhigSecurityCode--error, &.DhigSecurityCode--error:before":
    {
      borderColor: primitives.colors.accent.clay
    },
  '&.DhigSecurityCode--error[class*="MuiInputBase-root"]:after': {
    boxShadow: `0px 1px 0px 0px ${primitives.colors.accent.clay}`
  },
  "&.DhigSecurityCode:first-of-type": {
    borderLeft: `${primitives.borderWidths.medium} solid ${primitives.colors.tint["slate-450"]}`
  },
  "&.DhigSecurityCode:last-child": {
    borderRight: `${primitives.borderWidths.medium} solid ${primitives.colors.tint["slate-450"]}`
  },
  "&.DhigSecurityCode:last-child [class*='MuiDivider-root'], &.DhigSecurityCode:focus-within [class*='MuiDivider-root']":
    {
      display: "none"
    }
});

export default styles;
