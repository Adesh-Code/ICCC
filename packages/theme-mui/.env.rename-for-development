# To load the MUI X Pro license for local development, rename this file to .env and replace with the actual license string
DHIG_MUIXPRO_ENTRY=PASTE_MUI_X_PRO_LICENSE_STRING_HERE