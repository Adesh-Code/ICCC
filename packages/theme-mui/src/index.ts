// Themes
export { default as themeDark } from "./themes/dark";
export { default as themeLight } from "./themes/light";

// Utilities
export { configureClassnameGenerator } from "./classname-generator";
export { setLicenseKey } from "./set-license-key";
