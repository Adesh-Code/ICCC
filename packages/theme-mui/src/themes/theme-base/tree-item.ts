import type { ThemeData } from "../../types";

export const makeTreeItemStyles = ({
  primitives,
  tokens
}: ThemeData) => ({
  MuiTreeItem: {
    styleOverrides: {
      root: {
        color: tokens.colors["ink-on-background"]
      },
      content: {
        width: "unset",
        padding: primitives.spacings["4"],
        position: "relative",
        borderRadius: primitives.borderRadii.medium,
        zIndex: 1,
        ...tokens.transitions["state-blur"],
        "&:hover": {
          background: tokens.gradients["row-highlighted-hover-loud-reverse"],
          ...tokens.transitions["state-activate"]
        },
        "&.Mui-selected": {
          background: "none",
          ...tokens.transitions["state-activate"],
          "&:hover": {
            background:
              tokens.gradients["row-highlighted-hover-loud-reverse"],
            ...tokens.transitions["state-activate"]
          }
        },
        "&.Mui-focused": {
          ...tokens.transitions["state-activate"],
          boxShadow: tokens.effects["focus-ring-thin"],
          background: "none",
          "&:active": {
            boxShadow: "none",
            background:
              tokens.gradients["row-highlighted-pressed-loud-reverse"]
          }
        },
        "&.Mui-selected.Mui-focused": {
          background: "none",
          "&:active": {
            boxShadow: "none",
            background:
              tokens.gradients["row-highlighted-pressed-loud-reverse"]
          }
        },
        "&.Mui-disabled": {
          background: "none"
        },
        "&:active": {
          boxShadow: "none",
          background:
            tokens.gradients["row-highlighted-pressed-loud-reverse"],
          ...tokens.transitions["state-activate"]
        }
      },
      group: {
        margin: "0",
        position: "relative",
        padding: "0",
        "&:before": {
          content: '""',
          position: "absolute",
          height: "100%",
          width: "100%",
          bottom: 0,
          zIndex: 2,
          ...tokens.transitions["state-blur"],
          background: `linear-gradient(0deg, ${tokens.colors["fade-gradient-opaque"]} 0%, ${tokens.colors["fade-gradient-transparent"]} 100%)`
        },
        '&[class*="MuiCollapse-entered"]:before': {
          height: "0%"
        },
        '& [class*="MuiTreeItem-root"]': {
          marginLeft: primitives.spacings["4"]
        }
      },
      label: {
        paddingRight: primitives.spacings["4"]
      }
    }
  }
});
