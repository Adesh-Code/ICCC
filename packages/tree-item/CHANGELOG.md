# Change Log

All notable changes to this project will be documented in this file.
See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.

# [0.4.0](https://git.autodesk.com/dpe/iccc/compare/@iccc/tree-item@0.3.1...@iccc/tree-item@0.4.0) (2023-11-28)


### Features

* Enable theme splitting capability for ThemeMui ([#1055](https://git.autodesk.com/dpe/iccc/issues/1055)) ([c755947](https://git.autodesk.com/dpe/iccc/commits/c755947444b2bb9a71d2759cf4a2a6338d812f3d))


### BREAKING CHANGES

* üß® Removed ThemeMui component wrapper





## [0.3.1](https://git.autodesk.com/dpe/iccc/compare/@iccc/tree-item@0.3.0...@iccc/tree-item@0.3.1) (2023-11-10)


### Performance Improvements

* Cache build script with lerna6 and nx ([#1040](https://git.autodesk.com/dpe/iccc/issues/1040)) ([aa270cf](https://git.autodesk.com/dpe/iccc/commits/aa270cf478177941e61b7fcddc3437d21cf2fa27))





# [0.3.0](https://git.autodesk.com/dpe/iccc/compare/@iccc/tree-item@0.2.4...@iccc/tree-item@0.3.0) (2023-11-02)


### Features

* Upgrade to Lerna v6 and cache jest tests ([#1035](https://git.autodesk.com/dpe/iccc/issues/1035)) ([b5e8e89](https://git.autodesk.com/dpe/iccc/commits/b5e8e897ff49b37c04fcb261fe3a2bf7fa1cf727))





## [0.2.4](https://git.autodesk.com//dpe/iccc/compare/@iccc/tree-item@0.2.3...@iccc/tree-item@0.2.4) (2023-10-13)

**Note:** Version bump only for package @iccc/tree-item





## [0.2.3](https://git.autodesk.com//dpe/iccc/compare/@iccc/tree-item@0.2.2...@iccc/tree-item@0.2.3) (2023-08-07)

**Note:** Version bump only for package @iccc/tree-item





## [0.2.2](https://git.autodesk.com//dpe/iccc/compare/@iccc/tree-item@0.2.1...@iccc/tree-item@0.2.2) (2023-07-05)


### Bug Fixes

* button XD audit & loading icon color fix ([#854](https://git.autodesk.com//dpe/iccc/issues/854)) ([ebe903a](https://git.autodesk.com//dpe/iccc/commits/ebe903ad626e555bcd88adb6c818cdcae63c30cc))





## [0.2.1](https://git.autodesk.com//dpe/iccc/compare/@iccc/tree-item@0.2.0...@iccc/tree-item@0.2.1) (2023-07-03)

**Note:** Version bump only for package @iccc/tree-item





# [0.2.0](https://git.autodesk.com//dpe/iccc/compare/@iccc/tree-item@0.1.1...@iccc/tree-item@0.2.0) (2023-03-21)


### Features

* üé∏ dense variant for Tree view ([#683](https://git.autodesk.com//dpe/iccc/issues/683)) ([ebef634](https://git.autodesk.com//dpe/iccc/commits/ebef6349eb755a077c013d6003ed9249c8343cbc))





## [0.1.1](https://git.autodesk.com//dpe/iccc/compare/@iccc/tree-item@0.1.0...@iccc/tree-item@0.1.1) (2023-03-14)


### Bug Fixes

* üêõ Tree view border regression ([#695](https://git.autodesk.com//dpe/iccc/issues/695)) ([b648e6c](https://git.autodesk.com//dpe/iccc/commits/b648e6c722d30a799c576fd5832ed1c2b785b781))





# 0.1.0 (2023-03-13)


### Features

* implement flat style for Tree view component ([#663](https://git.autodesk.com//dpe/iccc/issues/663)) ([9b293c2](https://git.autodesk.com//dpe/iccc/commits/9b293c26f74624d33cddf01a9da921d86e9d63a9))
