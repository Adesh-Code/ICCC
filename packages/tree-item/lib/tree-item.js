"use strict";

function _typeof(o) { "@babel/helpers - typeof"; return _typeof = "function" == typeof Symbol && "symbol" == typeof Symbol.iterator ? function (o) { return typeof o; } : function (o) { return o && "function" == typeof Symbol && o.constructor === Symbol && o !== Symbol.prototype ? "symbol" : typeof o; }, _typeof(o); }
Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.FlatTreeItem = void 0;
Object.defineProperty(exports, "MinimalTreeItem", {
  enumerable: true,
  get: function get() {
    return _TreeItem.default;
  }
});
var _react = _interopRequireDefault(require("react"));
var _propTypes = _interopRequireDefault(require("prop-types"));
var _Divider = _interopRequireDefault(require("@mui/material/Divider"));
var _TreeItem = _interopRequireWildcard(require("@mui/lab/TreeItem"));
var _Typography = _interopRequireDefault(require("@mui/material/Typography"));
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function _getRequireWildcardCache(e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != _typeof(e) && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/*
 * Modified replica of MUI's internal component, TreeItemContent
 * https://github.com/mui/material-ui/blob/835e53eac76266ce37145a82365765190afc8709/packages/mui-lab/src/TreeItem/TreeItemContent.js
 * https://mui.com/material-ui/react-tree-view/#contentcomponent-prop
 */
var FlatContentComponent = /*#__PURE__*/_react.default.forwardRef(function FlatContentComponent(props, ref) {
  var classes = props.classes,
    className = props.className,
    label = props.label,
    nodeId = props.nodeId,
    iconProp = props.icon,
    expansionIcon = props.expansionIcon,
    displayIcon = props.displayIcon;
  var _useTreeItem = (0, _TreeItem.useTreeItem)(nodeId),
    disabled = _useTreeItem.disabled,
    expanded = _useTreeItem.expanded,
    selected = _useTreeItem.selected,
    focused = _useTreeItem.focused,
    handleExpansion = _useTreeItem.handleExpansion,
    handleSelection = _useTreeItem.handleSelection,
    preventSelection = _useTreeItem.preventSelection;
  var icon = iconProp || expansionIcon || displayIcon;
  var handleMouseDown = function handleMouseDown(event) {
    preventSelection(event);
  };
  var handleExpansionClick = function handleExpansionClick(event) {
    handleExpansion(event);
  };
  var handleSelectionClick = function handleSelectionClick(event) {
    handleSelection(event);
  };
  var classnames = className && className.split(" ") || [];
  classnames.push(classes.root);
  classnames.push("DhigTreeView--variant--flat");
  expanded && classnames.push(classes.expanded);
  selected && classnames.push(classes.selected);
  focused && classnames.push(classes.focused);
  disabled && classnames.push(classes.disabled);
  return /*#__PURE__*/_react.default.createElement("div", {
    className: "DhigTreeItem"
  }, /*#__PURE__*/_react.default.createElement(_Divider.default, {
    light: true
  }), /*#__PURE__*/_react.default.createElement("div", {
    className: classnames.join(" "),
    onMouseDown: handleMouseDown,
    onClick: handleExpansionClick,
    ref: ref
  }, /*#__PURE__*/_react.default.createElement("div", {
    className: classes.iconContainer
  }, icon), /*#__PURE__*/_react.default.createElement(_Typography.default, {
    onClick: handleSelectionClick,
    component: "div",
    className: classes.label
  }, label)));
});
FlatContentComponent.propTypes = {
  /**
   * Override or extend the styles applied to the component.
   */
  classes: _propTypes.default.object.isRequired,
  /**
   * className applied to the root element.
   */
  className: _propTypes.default.string,
  /**
   * The icon to display next to the tree node's label. Either a parent or end icon.
   */
  displayIcon: _propTypes.default.node,
  /**
   * The icon to display next to the tree node's label. Either an expansion or collapse icon.
   */
  expansionIcon: _propTypes.default.node,
  /**
   * The icon to display next to the tree node's label.
   */
  icon: _propTypes.default.node,
  /**
   * The tree node label.
   */
  label: _propTypes.default.node,
  /**
   * The id of the node.
   */
  nodeId: _propTypes.default.string.isRequired
};
var FlatTreeItem = exports.FlatTreeItem = function FlatTreeItem(props) {
  return /*#__PURE__*/_react.default.createElement(_TreeItem.default, _extends({
    ContentComponent: FlatContentComponent
  }, props));
};
FlatTreeItem.__docgenInfo = {
  "description": "",
  "methods": [],
  "displayName": "FlatTreeItem"
};