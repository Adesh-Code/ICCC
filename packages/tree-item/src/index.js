export { MinimalTreeItem, FlatTreeItem } from "./tree-item";
