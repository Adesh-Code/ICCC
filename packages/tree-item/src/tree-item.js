import React from "react";
import PropTypes from "prop-types";
import Divider from "@mui/material/Divider";
import MuiTreeItem, { useTreeItem } from "@mui/lab/TreeItem";
import Typography from "@mui/material/Typography";

/*
 * Modified replica of MUI's internal component, TreeItemContent
 * https://github.com/mui/material-ui/blob/835e53eac76266ce37145a82365765190afc8709/packages/mui-lab/src/TreeItem/TreeItemContent.js
 * https://mui.com/material-ui/react-tree-view/#contentcomponent-prop
 */
const FlatContentComponent = React.forwardRef(function FlatContentComponent(
  props,
  ref
) {
  const {
    classes,
    className,
    label,
    nodeId,
    icon: iconProp,
    expansionIcon,
    displayIcon
  } = props;

  const {
    disabled,
    expanded,
    selected,
    focused,
    handleExpansion,
    handleSelection,
    preventSelection
  } = useTreeItem(nodeId);

  const icon = iconProp || expansionIcon || displayIcon;

  const handleMouseDown = (event) => {
    preventSelection(event);
  };

  const handleExpansionClick = (event) => {
    handleExpansion(event);
  };

  const handleSelectionClick = (event) => {
    handleSelection(event);
  };

  const classnames = (className && className.split(" ")) || [];

  classnames.push(classes.root);
  classnames.push("DhigTreeView--variant--flat");
  expanded && classnames.push(classes.expanded);
  selected && classnames.push(classes.selected);
  focused && classnames.push(classes.focused);
  disabled && classnames.push(classes.disabled);

  return (
    <div className="DhigTreeItem">
      <Divider light />
      {/* eslint-disable-next-line jsx-a11y/click-events-have-key-events,jsx-a11y/no-static-element-interactions */}
      <div
        className={classnames.join(" ")}
        onMouseDown={handleMouseDown}
        onClick={handleExpansionClick}
        ref={ref}
      >
        <div className={classes.iconContainer}>{icon}</div>
        <Typography
          onClick={handleSelectionClick}
          component="div"
          className={classes.label}
        >
          {label}
        </Typography>
      </div>
    </div>
  );
});

FlatContentComponent.propTypes = {
  /**
   * Override or extend the styles applied to the component.
   */
  classes: PropTypes.object.isRequired,
  /**
   * className applied to the root element.
   */
  className: PropTypes.string,
  /**
   * The icon to display next to the tree node's label. Either a parent or end icon.
   */
  displayIcon: PropTypes.node,
  /**
   * The icon to display next to the tree node's label. Either an expansion or collapse icon.
   */
  expansionIcon: PropTypes.node,
  /**
   * The icon to display next to the tree node's label.
   */
  icon: PropTypes.node,
  /**
   * The tree node label.
   */
  label: PropTypes.node,
  /**
   * The id of the node.
   */
  nodeId: PropTypes.string.isRequired
};

export const FlatTreeItem = (props) => (
  <MuiTreeItem ContentComponent={FlatContentComponent} {...props} />
);

export { MuiTreeItem as MinimalTreeItem };
