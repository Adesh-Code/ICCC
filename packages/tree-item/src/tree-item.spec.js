import React from "react";
import { screen, render } from "@testing-library/react";
import userEvent from "@testing-library/user-event";
import TreeView from "@iccc/tree-view";
import { FlatTreeItem } from "./tree-item.js";

import { themeLight } from "@iccc/theme-mui";
import { createTheme, ThemeProvider } from "@mui/material/styles";

const light = createTheme(themeLight);

const arrange = ({ ...FlatTreeItemProps }) =>
  render(
    <ThemeProvider theme={light}>
      <TreeView variant="flat">
        <FlatTreeItem nodeId="1" label="Spreadsheets" {...FlatTreeItemProps}>
          <FlatTreeItem nodeId="2" label="Excel" {...FlatTreeItemProps} />
          <FlatTreeItem nodeId="3" label="Numbers" {...FlatTreeItemProps} />
        </FlatTreeItem>
      </TreeView>
    </ThemeProvider>
  );

describe("Flat Tree Item component", () => {
  it("renders", () => {
    const { container } = arrange();
    expect(container).toMatchSnapshot();
  });
  it("accepts classnames", () => {
    arrange({ className: "test-classname" });
    expect(screen.getByRole("treeitem")).toHaveClass("test-classname");
  });
  it("renders in a disabled state", () => {
    arrange({ disabled: true });
    expect(screen.getByRole("treeitem").firstChild.children[1]).toHaveClass(
      "Mui-disabled"
    );
  });
  it("expands when clicked", () => {
    arrange();
    const treeItemRoot = screen.getByRole("treeitem").firstChild.children[1];
    userEvent.click(treeItemRoot.children[1]);
    expect(treeItemRoot).toHaveClass(
      "Mui-expanded",
      "Mui-focused",
      "Mui-selected"
    );
  });
});
