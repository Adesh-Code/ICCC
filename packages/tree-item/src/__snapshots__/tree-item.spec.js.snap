// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Flat Tree Item component renders 1`] = `
<div>
  <ul
    aria-multiselectable="false"
    class="MuiTreeView-root DhigTreeView--variant--flat DhigTreeView--size--default emotion-0"
    id="mui-1"
    role="tree"
    tabindex="0"
  >
    <li
      aria-expanded="false"
      class="MuiTreeItem-root emotion-1"
      id="mui-1-1"
      role="treeitem"
      tabindex="-1"
    >
      <div
        class="DhigTreeItem"
      >
        <hr
          class="MuiDivider-root MuiDivider-fullWidth MuiDivider-light emotion-2"
        />
        <div
          class="emotion-3 MuiTreeItem-content DhigTreeView--variant--flat"
        >
          <div
            class="MuiTreeItem-iconContainer"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit emotion-4"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <svg
                fill="none"
                viewBox="0 0 6 16"
                xmlns="http://www.w3.org/2000/svg"
              >
                <g
                  id="caret-right-small"
                >
                  <path
                    d="M1.00098 4.5L4.50098 8L1.00098 11.5"
                    id="stroke"
                    stroke="currentColor"
                    stroke-linecap="round"
                    stroke-linejoin="round"
                    stroke-width="1.1"
                  />
                </g>
              </svg>
            </svg>
          </div>
          <div
            class="MuiTypography-root MuiTypography-body1 MuiTreeItem-label emotion-5"
          >
            Spreadsheets
          </div>
        </div>
      </div>
    </li>
  </ul>
</div>
`;
